var/spack/repos/builtin/packages/flecsale/package.py

```diff
@@ -55,7 +57,11 @@ def build_type(self):
             return 'Release'
 
     def cmake_args(self):
-        options = ['-DENABLE_UNIT_TESTS=ON']
+        options = [
+                '-DENABLE_UNIT_TESTS=ON'
+                '-DENABLE_OPENSSL=ON'
+                '-DENABLE_PYTHON=ON'
```

Comment:
These lines are causing the flake8 tests to fail. They should be indented by 4 spaces, not 8.